Btw, while testing this PR I found that some of the benchmark scripts are broken (for example `make time_stdlib_parse`):

```
The venv has been created in the ./venv directory
./venv/bin/python scripts/benchmark.py --parser=cpython --target=xxl parse
Traceback (most recent call last):
  File "/home/pablogsal/github/python/master/Tools/peg_generator/scripts/benchmark.py", line 17, in <module>
    from peg_extension import parse
ImportError: cannot import name 'parse' from 'peg_extension' (/home/pablogsal/github/python/master/Tools/peg_generator/peg_extension/__init__.py)
make: *** [Makefile:87: time_stdlib_parse] Error 1
```

given that we are focusing on the Python parser now, maybe we could adapt those to use the `_peg_parser` extension instead of trying to compile the parser.

--- a/Tools/peg_generator/Makefile
+++ b/Tools/peg_generator/Makefile
@@ -80,10 +80,10 @@ time_check: venv peg_extension/parse.c data/xxl.py
 
 time_stdlib: time_stdlib_compile
 
-time_stdlib_compile: venv data/xxl.py
+time_stdlib_compile: venv peg_extension/parse.c data/xxl.py
 	$(VENVPYTHON) scripts/benchmark.py --parser=cpython --target=xxl compile
 
-time_stdlib_parse: venv data/xxl.py
+time_stdlib_parse: venv peg_extension/parse.c data/xxl.py
 	$(VENVPYTHON) scripts/benchmark.py --parser=cpython --target=xxl parse
 
 test_local:

